### ISSUE TITLE
Add game to database

### ISSUE THREAD

**Issue by MrPiada** [franciscoBSalgueiro/en-croissant#191]
### Describe the feature

I'd like to expand the database of my games by adding some more but I can't find how to do it.
I tried from the analysis board but, as far as I understand, I can just copy-paste the pgn in the db files manually at the moment.

It would be great to have a button here "+" to add a game to the database:
<img width="1170" alt="Screenshot 2024-02-15 alle 09 22 18" src="https://github.com/franciscoBSalgueiro/en-croissant/assets/32706712/af4c2318-e94d-43e7-a37c-850cebaad294">

**Comment by SaggingRufus** [comment]
Was about to open a feature request for this!

**Comment by MrPiada** [comment]
Actually it would be sufficient to be able to save the game in an existing database (or update it whenever the database has a game with the same metadata) directly from an Analysis Board tab

**Comment by franciscoBSalgueiro** [comment]
With https://github.com/franciscoBSalgueiro/en-croissant/commit/eebf36a8bf4cd217737ff31a7b1250d22f056f95 it's now possible to add games to existing databases, as well as deleting them.

**Comment by ronaldsuwandi** [comment]
@franciscoBSalgueiro  Does this mean we can also update existing game in the database? Wasn't aware of this issue so I posted discussion #689

**Comment by franciscoBSalgueiro** [comment]
You can delete a game and add a new one as replacement. But the database format doesn't store annotations (that's something that I intend to change), so I'm not sure if that is useful to you yet.

**Comment by ronaldsuwandi** [comment]
right, that works but rather tedious especially if you have larger pgn and want to modify few games, having to go through game one by one to replace the correct one

storing annotations would definitely be useful :-) but not sure how complicated is it to implement
